errorHandlers/clientErrorHandler.js

```diff
@@ -0,0 +1,5 @@
+exports.clientErrorHandler = (err, req, res, next) => {
+  if (err.status === 400) res.status(400).send(err);
```

Comment:
Might also want to check for a message or msg key as well, just incase other errors also have a key of status.